### PR TITLE
Support multi-DAO

### PR DESCRIPTION
**Pull request by metaproph3t**
Allow multiple DAOs to leverage a single program. Addresses #75.

### CODE REVIEW

**Comment by Henry-E** [comment]
Are base lot size and twap expected value the only two configurable options that will be allowed when a DAO is initialized? 

What about the quote lot size, it's dumb but you do see a lot of meme coins with very small prices.
To avoid confusion, should expected price be passed in using the absolute value in USDC and then modified to price lots based on the base and quote lot sizes. Can it be trusted DAO creator won't mess this up. Depends on the UI probably.

Also it's taking away the PDA-ness of the DAO account. How much more annoying will this makes for things on the front end.

File: programs/autocrat_v0/src/lib.rs [hunk 1]
```diff
@@ -457,19 +473,13 @@ pub struct InitializeDAO<'info> {
...
     )]
     pub dao: Account<'info, DAO>,
     #[account(mut)]
     pub payer: Signer<'info>,
     pub system_program: Program<'info, System>,
-    #[account(mint::decimals = 9)]
-    pub meta_mint: Account<'info, Mint>,
+    pub token_mint: Account<'info, Mint>,
     #[account(mint::decimals = 6)]
     pub usdc_mint: Account<'info, Mint>,
```
Comment (Henry-E) [inline programs/autocrat_v0/src/lib.rs:484]: The pubkey of the USDC mint is known on mainnet and a cfg check could possibly be used to ensure that USDC is used and not another 6 decimals token. Not a massive deal though

**Comment by metaproph3t** [comment]
> Are base lot size and twap expected value the only two configurable options that will be allowed when a DAO is initialized?
> 
> What about the quote lot size, it's dumb but you do see a lot of meme coins with very small prices. To avoid confusion, should expected price be passed in using the absolute value in USDC and then modified to price lots based on the base and quote lot sizes. Can it be trusted DAO creator won't mess this up. Depends on the UI probably.
> 
> Also it's taking away the PDA-ness of the DAO account. How much more annoying will this makes for things on the front end.

Yeah, the UI needs to sort this out: we'd want it all to be configured in the base lot sizes

File: programs/autocrat_v0/src/lib.rs (hunk 1 above)
Comment (metaproph3t) [reply programs/autocrat_v0/src/lib.rs:484]: Yep, agreed. Let me create this as an issue.

**Comment by Henry-E** [comment]
The program ID here is `FuTPR6ScKMPHtZFwacq9qrtf9VjscawNEFTb2wSYr1gY` is that going to be the v0.3 devnet and mainnet program ID for the version of autocrat that supports multi-DAO?

**Pull request merged**